### PR TITLE
cmake update

### PR DESCRIPTION
**Pull request by partouf**
is still an rc though, maybe wait until full release?

### CODE REVIEW

**Comment by partouf** [comment]
rc3 is out, so not full release yet

**Comment by partouf** [comment]
Another note: https://www.kitware.com/cmake-3-28-0-rc3-is-ready-for-testing/
HIP seems to be now supported as well, not sure how that would work out, but good to know

**Comment by partouf** [comment]
https://www.kitware.com/cmake-3-28-0-rc4-is-ready-for-testing/ rc4 now, this might be a while...

File: bin/yaml/tools.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ tools:
     check_exe: bin/cmake --help
     symlink: cmake
     targets:
-      - 3.26.1
+      - 3.28.0-rc2
```
Comment (joblobob) [inline bin/yaml/tools.yaml:12]: As much as I would like to use the official modules support, and jump on one of the rc, would you rather prepare for the impending release and wait for the official release?

```suggestion
      - 3.28.0
```

Comment (partouf) [reply bin/yaml/tools.yaml]: thanks, yes, this is just a PR to remind me to regularly check the status of 3.28

**Review by mattgodbolt — APPROVED** [review]

**Comment by mattgodbolt** [comment]
lib.installation_context.FetchFailure: Fetch failure for https://github.com/Kitware/CMake/releases/download/v3.28.0/cmake-3.28.0-Linux-x86_64.tar.gz: <Response [404]>

**Comment by mattgodbolt** [comment]
Gonna revert and reopen. Sorry I really should have checked.

**Comment by mattgodbolt** [comment]
Opened https://github.com/compiler-explorer/infra/pull/1146

**Pull request merged**